packages/frames.js/src/core/createFrames.ts

```diff
@@ -58,8 +61,13 @@ export function createFrames<
     >([
       ...coreMiddleware,
       // @ts-expect-error hard to type internally so skipping for now
-      ...globalMiddleware,
+      imagesWorkerMiddleware({
+        imagesRoute: imagesRoute || "/",
```

Comment:
This breaks the ability to disable the image worker by passing `null` to `imagesRoute`.

Reply:
```suggestion
        imagesRoute: imagesRoute === null ? null : (imagesRoute || "/"),
```